cli/medperf/ui/cli.py

```diff
@@ -39,6 +41,7 @@ def print_warning(self, msg: str):
         self.__print(msg)
 
     def __print(self, msg: str = ""):
+        logging.info(msg)
```

Comment:
```suggestion
        logging.info("Console: " + str(msg))
```

Reply:
Btw about sensitive information. In debug logs we do already have a listed tree of `dataset` containment. May there be some sensitive information also?

Reply:
Removed logging for now